T3/Gui/OutputUi/FloatDictOutputUi.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Numerics;
+using System.Linq;
+using ImGuiNET;
+using T3.Core.Operator.Slots;
+using T3.Core.DataTypes;
+using T3.Gui;
+using Vector2 = System.Numerics.Vector2;
+
+namespace T3.Gui.OutputUi
+{
+    public class FloatDictOutputUi : OutputUi<Dict<float>>
+    {
+        public override IOutputUi Clone()
+        {
+            return new FloatDictOutputUi()
+            {
+                OutputDefinition = OutputDefinition,
+                PosOnCanvas = PosOnCanvas,
+                Size = Size
+            };
+        }
+
+        private static bool _autoFit = true;
+        private static bool _clear = false;
+        private static float _minFit = -1;
+        private static float _maxFit = 1;
+
+        private static List<List<float>> _previousValues;
```

Comment:
I'm not sure if this should be static. We should probably reset this automatically if, when switching between slots.

Reply:
Should not really be static. Either we should create more than one [FloatDictOutputUi](https://github.com/still-scene/t3/pull/140/files/1df7cf7a51b14d0e81d7152012c2aa7ccba34bae#diff-c16f25973d28ba49326a85486dc79916770be7eb913ef0a0de6da3c5eeae8354) instance, keep track of the specific dictionaries or reset on switching, yes.